src/goDebugConfiguration.ts

```diff
@@ -0,0 +1,88 @@
+import { execFile } from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as semver from 'semver';
+import * as vscode from 'vscode';
+
+import { getBinPathFromEnvVar } from './utils/pathUtils';
+
+const PLZ_MIN_VERSION = '16.1.0-beta.1';
+
+export class GoDebugConfigurationProvider implements vscode.DebugConfigurationProvider {
+    constructor(private plzBinPath: string = 'plz') { }
```

Comment:
`pleasew` probably makes more sense as a default. We can do that as a followup.

Reply:
I guess that's a more resilient approach to guarantee that please is/will be available, since that file is always(?) available in please repos?

Reply:
It's not always available but it's created when you `plz init` the repo. People could delete the file. I don't think it's unreasonable to require people put it there though.